frontend/components/pages/volunteer/index.tsx

```diff
@@ -0,0 +1,37 @@
+import Link from "next/link";
+import React from "react";
+import { IFaq } from "types/faq";
+import { IVolunteer } from "types/volunteer";
+import { FAQS } from "./faq";
+
+export const VolunteerPageComponent: React.FC<{ volunteerPageContent: IVolunteer[]; faqs: IFaq[] }> = ({
+  volunteerPageContent,
+  faqs,
+}) => {
+  return (
+    <>
+      {volunteerPageContent.map((volunteer) => (
+        <div key={volunteer.id}>
+          <h1 style={{ fontSize: "25px", fontWeight: "600", marginBottom: "30px" }}>{volunteer.Title}</h1>
+          <div dangerouslySetInnerHTML={{ __html: volunteer.Content }}></div>
+          <Link href={volunteer.ApplicationLink}>
```

Comment:
Request - Something I learnt recently was that nextjs `Link` components are only useful when linking to pages within the app, and not external content.

Could you replace this with a standard `<a href=...>` link instead? It would also be good if you could include `target="_blank" rel="noopener noreferrer"` in the element so that it opens in a new tab (target blank) and includes a safer rel (https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/)